roles/common/tasks/firewall.yml

```diff
@@ -24,4 +24,4 @@
     firewalld: service=https permanent=yes state=enabled
     notify: restart firewalld
 
-  when: fw_check|succeeded
+  when: fw_check is succeeded
```

Comment:
does it really work? The documentation says "is successful" or "is success"

Reply:
Yes, it works.
https://docs.ansible.com/ansible/2.5/user_guide/playbooks_tests.html#task-results